<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopPreflet</class>
 <widget class="QWidget" name="DesktopPreflet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Background &amp; Screen Saver</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabLauncher">
      <attribute name="title">
       <string>Launcher</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Icon &amp;Size</string>
           </property>
           <property name="buddy">
            <cstring>launcherIconSizeSpin</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="launcherIconSizeSlider">
           <property name="minimum">
            <number>32</number>
           </property>
           <property name="maximum">
            <number>96</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="launcherIconSizeSpin">
           <property name="minimum">
            <number>32</number>
           </property>
           <property name="maximum">
            <number>96</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabBackground">
      <attribute name="title">
       <string>Background</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListView" name="bgList">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>128</width>
           <height>80</height>
          </size>
         </property>
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
         <property name="layoutMode">
          <enum>QListView::Batched</enum>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&amp;Category:</string>
           </property>
           <property name="buddy">
            <cstring>bgCategory</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="bgCategory">
           <item>
            <property name="text">
             <string>Coatings</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Colors and Gradients</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>User's Selection</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&amp;Mode:</string>
           </property>
           <property name="buddy">
            <cstring>bgMode</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="bgMode">
           <item>
            <property name="text">
             <string>Fit to Screen</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fill Screen</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Stretch to Fit Screen</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Center</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tile</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabScreenSaver">
      <attribute name="title">
       <string>Screen Saver</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QListView" name="screensaverList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="screensaverPreview" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="screensaverOptions">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Options...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="screensaverTest">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Test</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>&amp;Start screensaver after:</string>
           </property>
           <property name="buddy">
            <cstring>screensaverTimeout</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="screensaverTimeout">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string>min</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>60</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabHotCorners">
      <attribute name="title">
       <string>Hot Corners</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>When the mouse pointer enters a corner:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QComboBox" name="corner1Combo">
           <item>
            <property name="text">
             <string>Do Nothing</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Show Workspaces</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Maximize Current Window</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Minimize Current Window</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Expose all Windows</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="corner2Combo">
           <item>
            <property name="text">
             <string>Do Nothing</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Show Workspaces</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Maximize Current Window</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Minimize Current Window</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Expose all Windows</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="desktopImage">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QComboBox" name="corner3Combo">
           <item>
            <property name="text">
             <string>Do Nothing</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Show Workspaces</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Maximize Current Window</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Minimize Current Window</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Expose all Windows</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="1">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="corner4Combo">
           <item>
            <property name="text">
             <string>Do Nothing</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Show Workspaces</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Maximize Current Window</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Minimize Current Window</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Expose all Windows</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
